infrastructure/kubernetes/mezo-production/monitoring/prometheus/configmap.yaml

```diff
@@ -35,3 +38,17 @@ data:
         metrics_path: /metrics
         static_configs:
           - targets: ['balance-exporter:9115', 'balance-exporter:9116']
+
+  alert_rules.yml: |
```

Comment:
For production we're configuring alerts in Grafana as it gives us more ways to manage the alert - see https://monitoring.mezo.org/grafana/alerting/list.
We should configure this alert in Grafana too, so we can define the messaging and destination.